docs/src/ScrollableContainer.doc.js

```diff
@@ -10,7 +10,7 @@ const card = (c) => cards.push(c);
 card(
   <PageHeader
     name="ScrollableContainer"
-    description="ScrollableContainer is used with anchored, hovering components such Flyout, Tooltip, Dropdown, or Typeahead. This is needed for proper positioning when the anchor component could scroll within the viewport."
+    description="ScrollableContainer is used with anchored components such Flyout, Tooltip, Dropdown, or Typeahead. A ScrollableContainer is needed for proper positioning when the Tooltip is anchored to an element that is located within a scrolling container. The use of ScrollableContainer ensures the Tooltip remains attached to its anchor when scrolling."
```

Comment:
no Oxford comma unfortunately (this kills me 😞)

Reply:
DONE